tests/e2e/defaultvmgroup_test.go

```diff
@@ -57,38 +64,100 @@ var _ = Suite(&DefaultVMGroupTestSuite{})
 
-	log.Printf("END: defaultvmgroup.TestReplaceDefaultTenantVMs")
+	misc.LogTestEnd(c.TestName())
+}
+
+// Verify volume creation after deleting _DEFAULT tenant
+// 1. Create a volume
+// 2. Verify volume from docker host and esx
+// 2. Delete default tenant
+// 3. Verify not able to create volume from docker host - Error: VM does not belong to any vmgroup
+// 4. Verify docker volume ls does not show any volumes
+// 5. Verify admin cli shows volume with VMGroup set as N/A
+// 6. Create _DEFAULT vmgroup with —default-datastore=_VM_DS
+// 7. Again create new volume - operation should succedd
+// 8. Verify volume from esx and docker host
+func (s *DefaultVMGroupTestSuite) TestDeleteDefaultVmgroup(c *C) {
+	misc.LogTestStart(c.TestName())
+	NoVgErrorMsg := "VolumeDriver.Create: VM " + s.config.DockerHostNames[0] + " does not belong to any vmgroup"
+
+	// Create a volume
+	out, err := dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable := verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeNames[0]))
+
+	// Verify if volume exists in default vmgroup
+	isVolInVmgroup := admincli.CheckVolumeExistInVmgroup(s.config.EsxHost, s.volumeNames[0], admincliconst.DefaultVMgroup)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Volume [%s] does not belong to vmgroup [%s]", s.volumeNames[0], admincliconst.DefaultVMgroup))
+
+	// Delete default vmgroup and verify it does not exist
+	admincli.DeleteVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup)
+
+	isVmgroupAvailable := admincli.CheckVmgroupAvailability(s.config.EsxHost, admincliconst.DefaultVMgroup)
+	c.Assert(isVmgroupAvailable, Equals, false, Commentf("Failed to delete vmgroup %s .", admincliconst.DefaultVMgroup))
+
+	// Create a volume - operation should fail as default vmgroup no longer exist
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[1])
+	c.Assert(err, Not(IsNil), Commentf(out))
+	c.Assert(strings.Contains(out, NoVgErrorMsg), Equals, true, Commentf(out))
+
+	// Verify existing volume - volumeName1 does not belong to any vmgroup (N/A)
+	isVolInVmgroup = admincli.CheckVolumeExistInVmgroup(s.config.EsxHost, s.volumeNames[0], NotAvailable)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Unexpected Behavior: Vmgroup for volume [%s] is not N/A. ", s.volumeNames[0]))
+
+	isVmgroupAvailable = admincli.CreateDefaultVmgroup(s.config.EsxHost)
+	c.Assert(isVmgroupAvailable, Equals, true, Commentf("Falied to create vmgroup %s .", admincliconst.DefaultVMgroup))
+
+	// Create a volume - operation should succeed as default vmgroup exists now
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[1])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
```

Comment:
need to add one more check here .. after recreation of default vmgroup .. created volume should be visible.